style.css

```diff
@@ -0,0 +1,360 @@
+    background-image: url(images/github_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 20px;
+    display: inline-block;
+    vertical-align: middle;
+    }
+
+.gh:hover:before {
+    content: "";
+    background-image: url(images/github_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 15px;
+    margin-left: 5px;
+    display: inline-block;
+    vertical-align: middle;
+    }
+
+.tel:hover:before {
+    content: "";
+    background-image: url(images/phone_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 15px;
+    margin-left: 5px;
+    display: inline-block;
+    vertical-align: middle;
+    } 
+    
+.ln:hover:before {
+    content: "";
+    background-image: url(images/linked_icon2.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 15px;
+    margin-left: 5px;
+    display: inline-block;
+    vertical-align: middle;
+    }
+
+.mail:hover:before {
+    content: "";
+    background-image: url(images/mail_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 15px;
+    margin-left: 5px;
+    display: inline-block;
+    vertical-align: middle;
+        }
+
+section {
+    padding: 10px 20px 10px 20px;
+}
+
+a {
+    text-decoration: none;
+    color: royalblue
+}
+
+footer {
+    text-align: center;
+    flex-direction: row;
+
+}
+
+.skills_lang {
+    background-color: #fde4cf;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+.education_courses {
+    background-color: #fde4cf;
+    padding-left: 20px;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+#link_list {
+    list-style-type: none;
+}
+
+.link_contaner {
+    margin-top: 20px;
+}
+
+.wrapper {
+    padding: 20px;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap-reverse;
+}
+
+footer {
+    padding-bottom: 10px;
+
+}
+
+.summary {
+    padding: 20px;
+}
+
+@media screen and (max-width: 1024px) {
+    
+
+}
+
+@media screen and (max-width: 996px) {
+    .skills_lang {
+        width: 100%;
+        flex-direction: row;
+        justify-content: space-evenly;
+        display: flex;
+        flex-wrap: wrap;
+        
+    } 
+    .Languages {
+        text-align: left;  
+    }
+    
+    .education_courses {
+        width: 100%;
+    }
+    
+}
+
+@media screen and (max-width: 500px) {
+    .name {
+        font-size: 20px;
+    }
+
+    h3 {
+        margin: 10px;
+
+    }
+
+    h2.position {
+        margin: 15px;
+        
+        margin-left: 20%;
+    }
+
+    .wrapper {
+        padding: 0;
+    }
+
+    p.summary {
```

Comment:
No need for tag usage. Class is enough probably? 
Same for the whole file.

Reply:
fix it, thank you